### DISCUSSION TITLE
How to run static-server using https ?

### DISCUSSION THREAD

**Discussion by wassim-ben-amor** [webdriverio/webdriverio#8109] [Q&A]
Hello,
I have noticed in the static-server that is using express here: https://github.com/webdriverio/webdriverio/blob/main/packages/wdio-static-server-service/src/launcher.ts
It is only launched using http.
Is there a way to create a server using a cert and key files to serve https://localhost:PORT using the wdio static-server?
Thank you,

**Comment by christian-bromann** [comment]
@wassim-ben-amor this is currently not possible but if you raise a PR for it I am happy to merge it 😉
